riscv/src/code_gen64.rs

```diff
@@ -0,0 +1,1507 @@
+use std::vec;
+
+use itertools::Itertools;
+use powdr_isa_utils::SingleDataValue;
+use powdr_isa_utils::{escape_label, quote};
+use powdr_number::KnownField;
+
+use crate::continuations::bootloader::{bootloader_and_shutdown_routine, bootloader_preamble};
+use crate::runtime::Runtime;
+use crate::runtime64::Runtime64;
+
+use crate::code_gen::{
+    InstructionArgs, MemEntry, Register, RiscVProgram, SourceFileInfo, Statement,
+};
+use crate::CompilerOptions;
+
+/// Translates a RISC-V program to POWDR ASM
+/// with constraints that work for a field >= the Goldilocks modulus.
```

Comment:
This file contains everything that was "removed" according to GH from the file above `code_gen.rs`, and I changed the text in this line. Everything else is untouched.